Readme.md

```diff
@@ -1,95 +1,97 @@
+
+This list does not say anything about the quality of either the listed or not listed [implementations](docs/implementations.md)!  
+The authors of the listed projects above are just known to work regularly together with LibSass developers.
+ 
 About
 -----
 
-LibSass is a C/C++ port of the Sass CSS precompiler. The original version was written in Ruby, but this version is meant for efficiency and portability.
-
-This library strives to be light, simple, and easy to build and integrate with a variety of platforms and languages.
+LibSass is a C++ port of the original Ruby Sass CSS compiler with a [C API](docs/api-doc.md).
+We coded LibSass with portability and efficiency in mind. You can expect LibSass to be magnitudes
+faster than Ruby Sass. We would even go as far and claim that it is the fastest CSS compiler
+around, which should be the case given it's written in C++.
 
 Developing
 ----------
 
-As you may have noticed, the LibSass repo itself has
-no executables and no tests. Oh noes! How can you develop???
+As noted above, the LibSass repository does not contain any binaries or other way to execute
+LibSass. Therefore, you need an implementer to develop LibSass. Easiest is to start with
+the official [SassC](http://github.com/sass/sassc) CLI wrapper. It is *guaranteed* to compile
+with the latest code in LibSass master, since it is also used in the CI process. There is no
+limitation here, as you may use any other LibSass implementer to test your LibSass branch!
 
-Well, luckily, [SassC](http://github.com/sass/sassc) is the official binary wrapper for
-LibSass and is *always* kept in sync. SassC is used by continous integration systems in
-LibSass repository. When developing LibSass, it is best to actually
-checkout SassC and develop in that directory with the SassC spec
-and tests there.
-
-We even run Travis tests for SassC!
-
-Tests
+Testing
 -------
 
-Since LibSass is a pure library, tests are run through the [SassSpec](https://github.com/sass/sass-spec) project using the [SassC](http://github.com/sass/sassc) driver.
+Since LibSass is a pure library, tests are run through the [Sass-Spec](https://github.com/sass/sass-spec)
+project using the [SassC](http://github.com/sass/sassc) CLI wrapper. To run the tests against LibSass while
+developing, you can run `./script/spec`. This will clone SassC and Sass-Spec under the project folder and
+then run the Sass-Spec test suite. You may want to update the clones to ensure you have the latest version.
+Note that the scripts in the `./script` folder are mainly intended for our CI needs.
 
-To run tests against LibSass while developing, you can run `./script/spec`. This will clone SassC and Sass-Spec under the project folder and then run the Sass-Spec test suite. You may want to update the clones to ensure you have the latest version.
+Building
+--------
 
-### DEBUG builds
+To build LibSass you need GCC 4.6+ or Clang/LLVM. If your OS is older, you may need to upgrade
```

Comment:
With clang it's hard to know. If you're on OSX (which is where clang is prevalent) it'll just work.